src/hello-world.md

```diff
@@ -37,4 +37,7 @@ Key points:
   the scope they are used in. Rust macros are actually only
   [partially hygienic](https://veykril.github.io/tlborm/decl-macros/minutiae/hygiene.html).
 
+* While it is not a functional language, it includes a range of
```

Comment:
I don't think we want to say that, we could make the point about any paradigm. Ie it's not OO but has aspects of object orientation

Reply:
You're right, but I've seen lots of people call Rust somewhat functional.

Could we include what you just wrote as well? 🙂 It's a common question in class! People like to try and classify Rust and it's neither a strict functional language nor is it a strict OO language.

Reply:
Yes maybe add OO as well? I'm fine with dropping functional again here as well, or just saying "Mutli-Paradigm"(as wikipedia does)

Reply:
I have added that, please take another look